server.go

```diff
@@ -90,11 +90,16 @@ func mutateHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var patch []patchOperation
+	var envVars []corev1.EnvVar
+
+	needsCreds := needsEnvVar(pod.Spec.Containers[0], "GOOGLE_APPLICATION_CREDENTIALS")
 
 	// Explicitly and silently exclude the kube-system namespace
 	if pod.ObjectMeta.Namespace != metav1.NamespaceSystem {
+		var v corev1.Volume
+		var mount corev1.VolumeMount
```

Comment:
I'm not against this in anyway, I'm purely just curious if there's a specific reason why you did this

Reply:
ah, i had a different refactor that required this, but then i removed it. fixed.